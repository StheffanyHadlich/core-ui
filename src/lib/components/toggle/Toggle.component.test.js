import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/toggle/__snapshots__"
});
