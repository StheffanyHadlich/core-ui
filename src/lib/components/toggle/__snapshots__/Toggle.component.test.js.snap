// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Toggle Default 1`] = `
<div>
  <div
    className="sc-bdVaJa gEyPxw"
  >
    <label
      className="switch sc-bwzfXH cycMUH"
    >
      <input
        checked={false}
        onChange={[Function]}
        type="checkbox"
      />
      <span
        className="slider"
      />
    </label>
    <span
      className="text sc-htpNat efAdPk"
    >
      Airplane Mode
    </span>
  </div>
  <div
    className="sc-bdVaJa gEyPxw"
  >
    <label
      className="switch sc-bwzfXH cycMUH"
    >
      <input
        checked={true}
        onChange={[Function]}
        type="checkbox"
      />
      <span
        className="slider"
      />
    </label>
    <span
      className="text sc-htpNat efAdPk"
    >
      Airplane Mode
    </span>
  </div>
</div>
`;

exports[`Storyshots Toggle ThemeProvider 1`] = `
<div>
  <div>
    <div
      className="sc-bdVaJa gEyPxw"
    >
      <label
        className="switch sc-bwzfXH oUJpL"
      >
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox"
        />
        <span
          className="slider"
        />
      </label>
      <span
        className="text sc-htpNat efAdPk"
      >
        Airplane Mode
      </span>
    </div>
    <div
      className="sc-bdVaJa gEyPxw"
    >
      <label
        className="switch sc-bwzfXH oUJpL"
      >
        <input
          checked={true}
          onChange={[Function]}
          type="checkbox"
        />
        <span
          className="slider"
        />
      </label>
      <span
        className="text sc-htpNat efAdPk"
      >
        Airplane Mode
      </span>
    </div>
  </div>
</div>
`;
